fuzz/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+
+[package]
+name = "lzxd-fuzz"
+version = "0.0.0"
+authors = ["Automatically generated"]
```

Comment:
Is there a reason to not include this file in the `.gitignore`? No idea how the tooling works here…

Reply:
This one is likely necessary - `cargo fuzz run lzxd` uses it.